src/main/histFileMod.F90

```diff
@@ -49,6 +49,7 @@ module histFileMod
   integer , public, parameter :: scale_type_strlen = 32 ! maximum number of characters for scale types
```

Comment:
```suggestion
  integer , private, parameter :: scale_type_strlen = 32 ! maximum number of characters for scale types
```
1) As far as I can tell this parameter could be declared private. 
2) The 3 public parameters above it get used outside of histFileMod. 
3) In the context of this PR, do I need to worry about the corresponding section of code in src/utils/clmfates_interfaceMod.F90 which looks at fincl statements for FATES vars? Follow up.